### DISCUSSION TITLE
Raudio / Moode 8.02

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#947] [General]
Hi, 

I recently made a benchmarking of RAudio vs Moose 8.02.
My audio system is a vintage upgraded high quality audiophile system based on QUAD amp and KEF 104/2 speakers.
That's said, it can play the sound quality difference between two sources with a good acuity. 

For me, definitively, the global sound quality : image, details, textures...is at the advantage of Raudio -> Very great job Rern !!

1 - Mid/Treble roll-off
I only notice a kind of roll-off on mid/treble bandwitch, specifically on voices. The "S" sounded "TCHHHHH". I tested it on many different HD release including DSD. I don't noticed this problem on Moode at all, just a few. 
May it can be investigate. 

2 - Cover display on external display
It will be interesting to have the choice of a simple cover display on. the main display, like the attached picture. 
![Capture d’écran 2022-06-16 à 12 58 21](https://user-images.githubusercontent.com/77485841/174057937-dd0a63ce-de66-4006-96fc-5fc1284428f4.jpg)

3 - DAC Filters
in the player settings in MOODE it's possible after selecting the proper Doc Hat driver, to selected the filter setting of this DAC in the Option menu. 
For ex for the 9038Q2M chipset :
* brick wall 
* corrected minimum phase fast 
* minimum phase slow 
* minimum phase fast 
* linear phase slow 
* linear phase fast 
* apodizing fast

Would it be possible to include this feature in the future. 

It's still a pleasure. to use Raudio, the ergonomy and other features are very well though.
Congrats to Rern, all the team and all people contributing to let this great distribution growing well.
Cheers

**Comment by kreativkod** [comment]
I could be wrong, but surely any difference in sound must be due to different DSP settings applied by the user. To my knowledge neither rAudio nor Moodeaudio applies any processing when you set them to play bitperfect. At that stage it is all down to your DAC.

**Reply by Rasb74** [reply]
The DSP is OFF, all is stetting on BIT PERFECT mode. I also think the DAC is the most responsible of the sound difference, but : How to explain sound difference between different distribution using the same RPI/DAC setup?

**Comment by R101G** [comment]
@Rasb74 

You don't say that the filter settings are the same, so might have answered your own question. Other than that, if both setups are bit perfect that just leaves the possibility of different RF noise interference from the Pi. You do not say how well isolated the DAC is, or whether it uses a recovered clock.

**Comment by Rasb74** [comment]
Interesting. I'm not sure the DAC is electrical isolated, in fact it is not ! 
The installation is as follow : RPI4 + Audiophonics Isabre 93038Q2M HAT + Linear power supply + 7" raspberry touch screen, all in the same box. 
Before this setup, the RPI+DAC were installed in a separate box, then the power supply was external. 
I'm very interesting, if you have some recommandation to upgrade this system. 
Thanks

**Comment by R101G** [comment]
@Rasb74 

There are far too many variables to suggest anything. Isolation could leave the Pi ungrounded, which in turn might allow it to radiate even more interference. Bit perfect is not maintained if the DAC input is asynchronous, and there might be timing differences between the operating systems.

I had problems with digital noise from a Pi 4 polluting the audio, and finally cured it by changing to a Pi 3 with a wired connection and all onboard wireless disabled.

**Comment by bbaba1886** [comment]
Hi Guys, I'm a cdrom and started Pi with Raudio since 2018. Same with Rasb74 testing Raudio 220526 vs moode 8.11 and Volumio  3.301. Found out the Raudio and moode has back ground noise like playing old LP disk (small Tik Tik Tik). The global sound quality was very good from Raudio and less Tik Tik noise compared with Moode. Only Volumio can provided clean back ground.

My system is PI3b with power filter (in metal case) + 1T (NVME / 2.5 harddisk with DSF / DFF files) + 5A linear power supply. 
Sound system is exd DA converter + audio research per-amp + Mcintosh power amp + ProAc speaker.

Any setting can get off the back ground noise.

Regards

**Comment by bbaba1886** [comment]
Hi, 
Tested the new version 227101 and small back ground noise still there.
Are their any setting can filter it out? Please advice.

Regards

**Reply by rern** [reply]
Try:
- `*.flac` or `*.mp3` files
- Headphones via 3.5mm on-board
- RPi + Power supply only
1. Power off
2. Disconnect everything (include the harddisk)
3. With rAudio on a new SD card
4. Connect only power supply and headphones (3.5mm on-board)
5. Listen for the noise while not playing
6. Copy the file to `/mnt/MPD/SD` > update Library database
7. Add and play the file

If the noise is still there, try another power supply.
If the noise is still there, try another RPi to rule out the RPi itself.

**Comment by bbaba1886** [comment]
Dear rern,

Thanks for your advice and back to my history on raudio website , noise had been found on my first download at year 2016 with v0.3 beta version. 
*.flac and *.mp3 files cannot play with my system. Sound come out like cassette tape. 
Never listen music with headphone. 
Got two sets of linear 5V 5A DC power supply. 
5 sets of PI3B and they are for my player project. ( 1 x for Moode, 1 x for PIcoreplayer, 1 x for Rune and 1 x for Volumio)
Over 10 pcs of 16G mini SD.
PI3B can provided very clean and quiet back ground while not playing. 
Friend of mine has the same issue with PI3 at their HiFi system as well.
I think maybe the small PI hardware not powerful enough to handle *.DSF or *.DFF files.

Anyway Thank You for the great afford and contribution (over 6 years) from the RuneAudio team.

Regards

**Reply by cYCL157** [reply]
i think rern was trying to guide you into diagnosis and hopefully finding the source of the problem, not telling you how to use radio going forward.

**Reply by bbaba1886** [reply]
I also wanted to diagnosis the problem as well.  Intrigued why no member reported on before. Maybe my gears having problem?  But Volumio working nicely with my HiFi gears.
